After installing add on, error appears when starting storybook

Hello, I am having trouble with the zeplin addon, I installed using the following command `npm install --save-dev storybook-zeplin`, then added it to the addons array, but when I run `npm run storybook` in my react application I get the following error:

```
✘ [ERROR] Could not resolve "vue/dist/vue"

    ../../node_modules/@zeplin/storybook-inspector/dist/index.js:1:20422:
      1 │ ...eact-element-to-jsx-string")},921:e=>{e.exports=require("vue/dist/vue")}},t={};return function n(o){var r=t[o];if(void 0!==r)...
        ╵                                                            ~~~~~~~~~~~~~~

  The module "./dist/vue" was not found on the file system:

    ../../node_modules/vue/package.json:32:16:
      32 │     "./dist/*": "./dist/*",
         ╵                 ~~~~~~~~~~

  Import from "vue/dist/vue.js" to get the file "../../node_modules/vue/dist/vue.js":

    ../../node_modules/@zeplin/storybook-inspector/dist/index.js:1:20435:
      1 │ ...lement-to-jsx-string")},921:e=>{e.exports=require("vue/dist/vue")}},t={};return function n(o){var r=t[o];if(void 0!==r)return...
        │                                                                   ^
        ╵                                                                   .js

  You can mark the path "vue/dist/vue" as external to exclude it from the bundle, which will remove
  this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.
  ```

I am not sure how to resolve this error, please advise. Thanks!

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "storybook-zeplin",
-    "version": "1.7.3",
+    "version": "1.7.13",
     "description": "Zeplin addon for Storybook",
     "main": "dist/index.js",
     "keywords": [
@@ -41,7 +41,7 @@
     },
     "dependencies": {
         "@zeplin/sdk": "^1.9.0",
-        "@zeplin/storybook-inspector": "^0.1.3",
+        "@zeplin/storybook-inspector": "^0.2.0",
         "query-string": "^6.12.1",
         "semver": "^7.3.5"
     },

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "storybook-zeplin",
-    "version": "1.7.13",
+    "version": "2.0.0",
     "description": "Zeplin addon for Storybook",
     "main": "dist/index.js",
     "keywords": [

